.github/ISSUE_TEMPLATE/BUG-REPORT.yml

```diff
@@ -0,0 +1,88 @@
+name: Bug report
+description: Report a bug in pylint
+labels: [bug, needs triage]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Please make sure that this issue wasn't already requested, or already implemented in the main branch.
+  - type: textarea
+    id: what-happened
+    attributes:
+      label: What is the bug about ?
+      description:
+        Please provide the code you linted, and configurations files and multiple named
+        file if required
```

Comment:
We should make it as simple as possible -> shouldn't ask three things with one question. It's probably enough to ask for code that is causing the issue here.

As most errors are easily reproduceable with the default config, we should be fine adding an optional field later one for it.